src/full/Agda/TypeChecking/Monad/Base.hs

```diff
@@ -4963,3 +5329,5 @@ instance NFData UnificationFailure
 instance NFData UnquoteError
 instance NFData TypeError
 instance NFData LHSOrPatSyn
+
+-- -}
```

Comment:
This could be cleaned up, if I'm allowed to nitpick.

Reply:
Fixed that.